src/datreant/core/tests/test_treants.py

```diff
@@ -26,6 +27,15 @@ def treant(self, tmpdir):
             c = dtr.treants.Treant(TestTreant.treantname)
         return c
 
+    @pytest.fixture
+    def basic_treant(self, tmpdir, treantclass):
+        # treant with tags and cats, in tmpdir
+        with tmpdir.as_cwd():
+            t1 = treantclass('Rincewind')
+            t1.tags.add('magical')
+            t1.categories.add({'colour': 'octarine'})
+            yield t1
```

Comment:
How does it not work? It's just this: http://doc.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code